pandas/tests/copy_view/test_chained_assignment_deprecation.py

```diff
@@ -45,12 +45,9 @@ def test_methods_iloc_warn(using_copy_on_write):
 def test_methods_iloc_getitem_item_cache(
     func, args, using_copy_on_write, warn_copy_on_write
 ):
-    df = DataFrame({"a": [1.5, 2, 3], "b": 1.5})
+    df = DataFrame({"a": [1, 2, 3], "b": 1})
     ser = df.iloc[:, 0]
-    # TODO(CoW-warn) should warn about updating a view for all methods
-    with tm.assert_cow_warning(
-        warn_copy_on_write and func not in ("replace", "fillna")
-    ):
+    with tm.assert_cow_warning(warn_copy_on_write and func == "replace"):
         getattr(ser, func)(*args, inplace=True)
```

Comment:
This is a bit fishy. The interpolate PR changed that to floats, this changes it back. But in the end this test is for ensuring there is no chained assignment warning, not about whether there is an "setting value on view" warning, will clean-up in a follow-up PR.